### PR TITLE
Adds chirurgiens-dentistes-en-france.fr to private domains

### PR DESCRIPTION
**Pull request by yaslama**
P.P.COM (ppcom.fr via denti.site) offers our dentist customers in France a website hosted in a domain *.chirurgiens-dentistes-en-france.fr.
Specifically, each dentist has a domain in the form of dr-<firstname>-<lastname>.chirurgiens-dentistes-en-france.fr
E.g.: dr-dupond-pierre.chirurgiens-dentistes-en-france.fr
One can see other examples of sites in denti.site

Reason for inclusion

Cookie isolation. Each dentist can partially modify himself his site so cookie isolation is needed for security reason.
We could also get rid of our wildchar certificate and obtain easily (using Let's Encrypt for instance) a certificate per active website

### CODE REVIEW

**Comment by yaslama** [comment]
FYI, I added the TXT record in the dns:
```
$ host -t TXT  _psl.chirurgiens-dentistes-en-france.fr
_psl.chirurgiens-dentistes-en-france.fr descriptive text "https://github.com/publicsuffix/list/pull/177"
```

**Comment by yaslama** [comment]
I just rebased the PR. Can you please review it?

**Comment by yaslama** [comment]
Hi @weppos 

I see you already merge #200 
What is missing in #177 in order to be merged ?
Thanks a lot.

**Comment by sleevi** [comment]
Sorry for the delay @yaslama 

Can you please update this patch to follow the [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines)

Specifically:
* Rebase to master
* Correct your header to remove the abbreviation, including only the full company name
* Correct your header to include a proper URL (e.g. `https://ppcom.fr` )
* Correct your header to include a space between the above two
* Correct your header to ensure that Submitted is capitalized

I have verified the TXT record is correct.

**Comment by yaslama** [comment]
Hi @sleevi 

Thanks for your comments and your quick answer. I just made the modifications you asked.
--yas

**Comment by sleevi** [comment]
You're missing the colon between the name and URL in the header (see the Guidelines) and then it should be good to merge.

**Comment by yaslama** [comment]
Hi @sleevi I added the colon. I think it's ok now. Thanks again

File: public_suffix_list.dat [hunk 1]
```diff
@@ -11338,6 +11338,10 @@ gotpantheon.com
 // Submitted by Steve Leung <steveleung@peplink.com>
 mypep.link
 
+// Publication Presse Communication SARL : https://ppcom.fr
```
Comment (sleevi) [inline public_suffix_list.dat]: You need to move this down two entries (Pu > Pr) - Line 1348. So sorry this wasn't caught when you fixed the name.

**Comment by yaslama** [comment]
Hi @sleevi 
Sorry for the mistake. I just fixed it.

**Pull request merged**